src/test/java/edu/kit/ipd/crowdcontrol/objectservice/quality/ratingQuality/RatingQualityByDistributionTest.java

```diff
@@ -130,4 +131,10 @@ public void throwExceptionMissingRatings(){
         identifier.identifyRatingQuality(ratings,params,maxQuality, minQuality);
 
     }
+    @Test
+    public void testGetter(){
+        assertEquals(identifier.getAlgorithmName(),RatingQualityByDistribution.ALGORITHM_NAME);
+        assertEquals(identifier.getAlgorithmDescription(),RatingQualityByDistribution.ALGORITHM_DESCRIPTION);
+        assertTrue(identifier.getParams() != null);
```

Comment:
`assertNotNull` is better, because it provides useful output if the assertion fails.